awx/main/managers.py

```diff
@@ -243,7 +243,15 @@ def capacity_values(self, qs=None, tasks=None, breakdown=False, graph=None):
         for t in tasks:
             # TODO: dock capacity for isolated job management tasks running in queue
             impact = t.task_impact
-            if t.status == 'waiting' or not t.execution_node:
+            control_groups = []
+            if hasattr(t, 'controller_node') and t.controller_node:
```

Comment:
The use of `hasattr` is a little suspicious here. Should we not just define a default property on `UnifiedJob`?

Reply:
It looks like it's a database field on UnifiedJob: https://github.com/ansible/awx/blob/ecf9a0827dc29338493ea7431393913029785458/awx/main/models/unified_jobs.py#L586

Why do we need this `hasattr`?

Reply:
I ran into an exception in this area when working on this in my previous PR...but probably just me thrashing. I'll just call `t.controller_node`

Reply:
done